meta-lmp-base/recipes-bsp/u-boot/u-boot-fio_2022.04.bb

```diff
@@ -1,5 +1,5 @@
 require u-boot-fio-common.inc
 
-SRCREV = "fd9b3d3ed417b954a714bd97ae47f2e2aeac8a3f"
+SRCREV = "08452551bfa6fad7af475757be88d9cb0879eeff"
```

Comment:
Do we have the same changes in our imx fork?

Reply:
not yet. I was planning to do that in different PR, as in this one I enable support only for stm32mp15

Reply:
sure, works for me